<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">jdEolConverter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>jdEolConverter converts the End of Line of all files in the given Directory. Select a Directory and click OK to get started.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>End of Line</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rad_unix">
        <property name="text">
         <string notr="true">Unix (LF)</string>
        </property>
        <property name="shortcut">
         <string notr="true"/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rad_windows">
        <property name="text">
         <string notr="true">Windows (CRLF)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rad_mac">
        <property name="text">
         <string notr="true">Mac (CR)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chk_regex">
        <property name="text">
         <string>Convert only files which name matches this RegEx</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_regex">
          <property name="text">
           <string>RegEx:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_regex"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_skip_hidden">
        <property name="text">
         <string>Skip hidden files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_recursive">
        <property name="text">
         <string>Recursive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
